amundsen_application/static/js/components/AvatarLabelList/styles.scss

```diff
@@ -0,0 +1,152 @@
+@import 'variables';
+
+.editable-list-component {
+  .btn.add-list-item {
+    color: $gray-light;
+    padding: 4px 1px;
+    -webkit-box-shadow: none !important;
+    box-shadow: none !important;
+    text-align: left;
+    height: 32px;
+
+    &:focus,
+    &:not(.disabled):hover,
+    &:not([disabled]):hover {
+      color: $gradient-4;
+      background-color: transparent;
+      border-color: transparent;
+
+      .icon {
+        background-color: $gradient-4;
+      }
+    }
+  }
```

Comment:
I think we should define a new type of button where the button is essentially an icon. We only had one example of this button with the old `edit-button`, but it looks like we have a few more added here. 

Perhaps we could define a `icon-button` or `flat-icon-button` in `_buttons-default.scss` and give it the transparent background/border/shadow/etc. Then wherever we want to use it it can be as simple as 

```
<button className="btn icon-button">
  <img className="icon icon-circle-plus"/>
</button>
```

Same applies for delete

Reply:
Mind if I do that in a separate button-focused PR, where we take the 3 button styles that will get reused from this component and define them for the whole app? Notice the overrides I had to add for the `Cancel` and `Save` button here. These will also get reused in all of our existing and future editing modals.

Reply:
Let's put a TODO here and file a ticket.